api/test/json-api.e2e-spec.ts

```diff
@@ -3,9 +3,22 @@ import { INestApplication } from '@nestjs/common';
 import * as request from 'supertest';
 import { AppModule } from '../src/app.module';
 import { omit } from 'lodash';
+import { E2E_CONFIG } from './e2e.config';
+import { ProjectsTestUtils } from './utils/projects.test.utils';
+import { Organization } from 'modules/organizations/organization.api.entity';
+import { OrganizationsTestUtils } from './utils/organizations.test.utils';
+import * as JSONAPISerializer from 'jsonapi-serializer';
+import { Project } from 'modules/projects/project.api.entity';
 
 describe('JSON API Specs (e2e)', () => {
   let app: INestApplication;
+  let jwtToken: string;
+  let fakeCountry: string = 'ESP';
```

Comment:
`const fakeCountry = 'ESP'`. TS compiler complains ; )

I think we should enable "build" step in CI, otherwise we may end up in broken `develop` (in fact this is e2e so it isn't that important there but in general, we should)

Reply:
Thanks for pointing out. It does not complain to me, but I'll add this change.
Maybe we should start using WebStorm...? xD

Reply:
Oh, all right, is actually ESLint complaining, and I know why didn't complain to be. My bad here